From 9c19df9787b765fadf9d35004f18526edcaa5a9e Mon Sep 17 00:00:00 2001
From: Hisashi Nakamura <hisashi.nakamura.ak@renesas.com>
Date: Wed, 12 Jun 2013 15:41:45 +0900
Subject: [PATCH 486/715] ARM: shmobile: lager: Fix the clock frequency of
 arch_timer in DTS

The frequency of the arch_timer is changed from 13MHz to 10MHz.

Signed-off-by: Hisashi Nakamura <hisashi.nakamura.ak@renesas.com>
(cherry picked from commit 654e99ded09eaae1de31d0294506cdf72a85126f)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 arch/arm/boot/dts/r8a7790.dtsi      |    1 +
 arch/arm/configs/shmobile_defconfig |    2 ++
 2 files changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7790.dtsi b/arch/arm/boot/dts/r8a7790.dtsi
index e37aefb..0b91ff9 100644
--- a/arch/arm/boot/dts/r8a7790.dtsi
+++ b/arch/arm/boot/dts/r8a7790.dtsi
@@ -87,6 +87,7 @@
 				<1 14 0xf08>,
 				<1 11 0xf08>,
 				<1 10 0xf08>;
+		clock-frequency = <10000000>;
 	};
 
 	irqc0: interrupt-controller@e61c0000 {
diff --git a/arch/arm/configs/shmobile_defconfig b/arch/arm/configs/shmobile_defconfig
index 8dc551f..13010ef 100644
--- a/arch/arm/configs/shmobile_defconfig
+++ b/arch/arm/configs/shmobile_defconfig
@@ -14,7 +14,9 @@ CONFIG_SLAB=y
 CONFIG_ARCH_SHMOBILE=y
 CONFIG_ARCH_R8A7790=y
 CONFIG_MACH_LAGER=y
+CONFIG_SHMOBILE_TIMER_HZ=100
 # CONFIG_SH_TIMER_TMU is not set
+# CONFIG_EM_TIMER_STI is not set
 # CONFIG_SWP_EMULATE is not set
 CONFIG_ARM_ERRATA_430973=y
 CONFIG_ARM_ERRATA_458693=y
-- 
1.7.10.4

